tools/get_dicom.pl

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/perl -w 
+use strict;
+
+=pod
+
+=head1 NAME
+
+get_dicom.pl - extracts DICOM files for specific patient names/scan types
+
+=head1 SYNOPSIS
+
+perl get_dicom.pl [-p profile] [-n patient_name_patterns] [-t scan_type_patterns] [-d tmp_dir] [-o tarBasename]
```

Comment:
rename script here if you agree with my above suggestion.

Reply:
I agree: renamed.